docs/topics/http/views.txt

```diff
@@ -166,3 +166,40 @@ The :func:`~django.views.defaults.bad_request` view is overridden by
 
     Use the :setting:`CSRF_FAILURE_VIEW` setting to override the CSRF error
     view.
+
+Testing custom error views
+--------------------------
+
+To test the response of your custom error handler, you should raise an
+exception in the view function. Here is an example::
+
+    from django.conf.urls import url
+    from django.core.exceptions import PermissionDenied
+    from django.http import HttpResponse
+    from django.test import SimpleTestCase, override_settings
+
+
+    def response_error_handler(request, exception=None):
+	return HttpResponse('Error handler content', status=403)
+
+
+    # Raise the exception in your view:
+    def permission_denied_view(request):
+	raise PermissionDenied
+
+
+    urlpatterns = [
+	url(r'^$', permission_denied_view),
```

Comment:
Use django's new `url.path` https://docs.djangoproject.com/en/2.1/ref/urls/#django.urls.path and make it e.g. `/403/`